Bottom: 19e1b2d5daefb222ba1b4b8c7d9d406d6a6ab2fc
Top:    12e3b3848c4ed13c01872bfe7e73c34224e5f2e6
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-23 10:51:51 +0800

vhost: bypass wrap counter in SET/GET_VRING_BASE

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 112f680..500ee47 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -1415,10 +1415,12 @@ long vhost_vring_ioctl(struct vhost_dev *d, unsigned int ioctl, void __user *arg
 			break;
 		}
 		vq->last_avail_idx = s.num;
+#if 0
 		if (vhost_has_feature(vq, VIRTIO_F_RING_PACKED)) {
 			vq->last_avail_wrap_counter = s.num >> 31;
 			vq->avail_wrap_counter = vq->last_avail_wrap_counter;
 		}
+#endif
 		/* Forget the cached index value. */
 		vq->avail_idx = vq->last_avail_idx;
 		break;
